docs/installing-open-mpi/required-support-libraries.rst

```diff
@@ -3,30 +3,38 @@
 Required support libraries
 ==========================
 
-Open MPI uses the following support libraries:
-
-#. `Hardware Locality (hwloc)
-   <https://www.open-mpi.org/projects/hwloc/>`_: This library is
-   required; Open MPI will not build without it.
-
-#. `Libevent <https://libevent.org/>`_: This library is required; Open
-   MPI will not build without it.
-
-#. `PMIx <https://pmix.org/>`_: This library is required; Open MPI
-   will not build without it.
-
-#. `PRRTE <https://github.com/openpmix/prrte>`_: This library is
-   optional in some environments.  PRRTE provides Open MPI's
-   full-featured ``mpirun`` / ``mpiexec`` MPI application launchers
-   (the two are identical; they are symbolic links to the same executable).
-
-   * If your environment uses another MPI application launcher
-     (e.g., Slurm users can use the ``srun`` launcher to "direct
-     launch" Open MPI applications), then the use of PRRTE is
-     optional.
-   * If your environment has no other MPI application launcher, then
-     you need to install PRRTE and build Open MPI with PRRTE
-     support.
+Open MPI requires the following support libraries with the minimum listed versions:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 10 25
+
+   * - Library
+     - Version
+     - Notes
+   * - `Hardware Locality <https://www.open-mpi.org/projects/hwloc/>`_
+     - |hwloc_min_version|
+     - | This library is required; Open MPI will not build without it.
+   * - `Libevent <https://libevent.org/>`_
+     - |event_min_version|
+     - | This library is required; Open MPI will not build without it.
+   * - `PMIx <https://pmix.org/>`_
+     - |pmix_min_version|
+     - | This library is required; Open MPI will not build without it.
+   * - `PRRTE <https://github.com/openpmix/prrte>`_
+     - |prte_min_version|
+     - | This library is optional in some environments. PRRTE provides
+       | Open MPI's full-featured ``mpirun`` / ``mpiexec`` MPI
+       | application launchers (the two are identical; they are symbolic
+       | links to the same executable).
+
+       * | If your environment uses another MPI application launcher
+         | (e.g., Slurm users can use the ``srun`` launcher to "direct
+         | launch" Open MPI applications), then the use of PRRTE is
+         | optional.
+       * | If your environment has no other MPI application launcher,
+         | then you need to install PRRTE and build Open MPI with PRRTE
+         | support.
```

Comment:
I'd word this a little differently: you could read this bullet to imply that a user must install PRTE separately before installing Open MPI.

Reply:
Can you keep the original sentence, too?  "If your environment has no other MPI application launcher, then you need to install PRRTE and build Open MPI with PRRTE support."  I was suggesting adding the new sentences *after* the original sentence

Reply:
Shoot, I thought I did. Ugh, will add it back. Thanks for catching

Reply:
Added it back.